library/swift/test/ElementTests.swift

```diff
@@ -0,0 +1,14 @@
+import Envoy
+import XCTest
+
+final class ElementTests: XCTestCase {
+    func testElementNotEqualToObjectOfUnrelatedType() {
+        let element = Element("foo")
+        XCTAssertFalse(element.isEqual("bar"))
+    }
+
+    func testElementsEquality() {
+      XCTAssertEqual(Element("foo"), Element("foo"))
+      XCTAssertNotEqual(Element("foo"), Element("bar"))
```

Comment:
Should we explicitly do an `isEqual` check here too?

Reply:
maybe lets add 2 additional assertions - they would use `isEqual` method. WDYT?

Reply:
sure